### ISSUE TITLE
useWindowAsScrollContainer example

### ISSUE THREAD

**Issue by mike623** [seatgeek/react-infinite#198]
fail to fire `onInfiniteLoad` when useWindowAsScrollContainer and scroll to bottom. Is there any useWindowAsScrollContainer example can be reference?

**Comment by Nisthar** [comment]
Yes, `onInfiniteLoad` only fires on the first load and does not fire on scrolling to the bottom. Any solutions for this? Is this supposed to be like this?

**Comment by acomito** [comment]
same issue

**Comment by Tim152** [comment]
same issue

**Comment by garetht** [comment]
This is a [working example](https://github.com/seatgeek/react-infinite/blob/master/examples/borderless-window.jsx) with `useWindowAsScrollContainer`. Does that still happen for you on that example?

**Comment by fakiolinho** [comment]
Ended up here because of the same issue. Checked that working example too without any luck

**Comment by jrmienna** [comment]
+1

**Comment by christianmurlowski** [comment]
I have this issue too :(

**Comment by garetht** [comment]
What browser are you on, and did you try the example in the examples folder?
